app/views/layouts/_navbar.html.haml

```diff
@@ -1,4 +1,4 @@
-%nav.top-bar{"data-topbar" => "", :role => "navigation"}
+.top-bar{"data-topbar" => "", :role => "navigation"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace